pilot/cmd/pilot-agent/main.go

```diff
@@ -133,6 +134,7 @@ var (
 			proxyConfig.ConnectTimeout = ptypes.DurationProto(connectTimeout)
 			proxyConfig.StatsdUdpAddress = statsdUDPAddress
 			proxyConfig.ProxyAdminPort = int32(proxyAdminPort)
+			proxyConfig.Concurrency = int32(concurrency)
```

Comment:
I'm not sure how this is going to work - setting the concurrency in the global mesh config will not work well, different pods may have different settings, based on load / CPU allocation.

Reply:
@costinm I agree, this is an issue. We only need to expose it when it's running through pilot agent.